src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separ = fromFormat[3];
+  const dateArr = date.split(separ);
+  const fromDate = fromFormat.slice(0, -1);
+  const futureSepar = toFormat[3];
+  const toDate = toFormat.slice(0, -1);
```

Comment:
there is one more approach, you can use an object to save your date parts (DD, YY, YYYY, MM), and use them.
Your solution is fine, just can be a bit simplified, if you want you can try to solve this in another way